pkg/kubefedctl/util/util.go

```diff
@@ -118,6 +121,22 @@ func HealthCheckRoleName(serviceAccountName, namespace string) string {
 	return fmt.Sprintf("kubefed-controller-manager:%s:healthcheck-%s", namespace, serviceAccountName)
 }
 
+// Checks if a resource with the given Kind and group is a Federated
 func IsFederatedAPIResource(kind, group string) bool {
 	return strings.HasPrefix(kind, FederatedKindPrefix) && group == options.DefaultFederatedGroup
 }
+
+// GetNamespace returns namespace of the current context
+func GetNamespace(hostClusterContext string, kubeconfig string, config FedConfig) (string, error) {
+
```

Comment:
Please remove this unnecessary linebreak. I only expect to see a separation between the signature and the first line of the function/method would be if the signature spanned multiple lines such that the result of gofmt made it difficult to differentiate between the signature and the first line.

Reply:
Done